.vscode/settings.json

```diff
@@ -1,3 +1,9 @@
 {
-    "editor.insertSpaces": false
+	"editor.tabSize": 2, // Set the number of spaces per tab
```

Comment:
Didn't we say we won't commit settings?

Reply:
Yep I updated this part and remove the settings.json totally.